From: Kill Your TV <killyourtv@i2pmail.org>
Date: Fri, 3 Oct 2014 16:47:20 +0000
Subject: fix makefile

---
 config.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/config.mk b/config.mk
index 49bf5b6..8053af1 100644
--- a/config.mk
+++ b/config.mk
@@ -2,11 +2,11 @@
 VERSION = 0.1
 
 # paths
-PREFIX = /usr/local
+PREFIX = /usr
 MANPREFIX = $(PREFIX)/share/man
 
 CC = cc
 LD = $(CC)
 CPPFLAGS = -DVERSION=\"${VERSION}\"
-CFLAGS   = -I/usr/local/include -Wall -Wunused $(CPPFLAGS)
-LDFLAGS  = -L/usr/local/lib -ltoxcore -ltoxav -ltoxencryptsave
+CFLAGS   = -I/usr/include -Wall -Wunused $(CPPFLAGS) $(shell dpkg-buildflags --get CFLAGS)
+LDFLAGS  = -L/usr/lib -ltoxcore -ltoxav -ltoxencryptsave $(shell dpkg-buildflags --get LDFLAGS)
