From: Kill Your TV <killyourtv@i2pmail.org>
Date: Wed, 8 Oct 2014 12:27:47 +0000
Subject: kfreebsd fix

---
 ratox.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/ratox.c b/ratox.c
index f6c0c25..cb4c23b 100644
--- a/ratox.c
+++ b/ratox.c
@@ -26,6 +26,10 @@
 #include "readpassphrase.h"
 #include "util.h"
 
+#ifndef O_DIRECTORY
+#define O_DIRECTORY 0
+#endif
+
 const char *reqerr[] = {
 	[-TOX_FAERR_TOOLONG]      = "Message is too long",
 	[-TOX_FAERR_NOMESSAGE]    = "Please add a message to your request",
