From: Kill Your TV <killyourtv@i2pmail.org>
Date: Wed, 8 Oct 2014 15:29:58 +0000
Subject: fix compilation on raspberrypi

---
 config.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/config.mk
+++ b/config.mk
@@ -9,4 +9,4 @@
 LD = $(CC)
 CPPFLAGS = -DVERSION=\"${VERSION}\"
 CFLAGS   = -g -I/usr/include -Wall -Wunused $(CPPFLAGS) $(shell dpkg-buildflags --get CFLAGS)
-LDFLAGS  = -g -L/usr/lib -ltoxcore -ltoxav -ltoxencryptsave $(shell dpkg-buildflags --get LDFLAGS)
+LDFLAGS  = -g -L/usr/lib -ltoxcore -ltoxav -ltoxencryptsave -lrt $(shell dpkg-buildflags --get LDFLAGS)
