From: Kill Your TV <killyourtv@i2pmail.org>
Date: Sat, 11 Oct 2014 14:47:05 +0000
Subject: fix build with new toxcore

---
 ratox.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ratox.c b/ratox.c
index 9fcf491..dd58959 100644
--- a/ratox.c
+++ b/ratox.c
@@ -1070,7 +1070,7 @@ dataload(void)
 	if (read(fd, data, sz) != sz)
 		eprintf("read %s:", DATAFILE);
 
-	if (tox_is_data_encrypted(data) == 1) {
+	if (tox_is_save_encrypted(data) == 1) {
 		if (encryptdatafile == 0)
 			printout("Data : %s > Encrypted, but saving unencrypted\n", DATAFILE);
 		while (readpass("Data : Passphrase > ") < 0 ||
